bin/sl-meshctl.js

```diff
@@ -92,11 +90,11 @@ if (process.env.SSH_PORT) {
 dieIf.url = apiUrl;
 maybeTunnel(apiUrl, sshOpts, function(err, apiUrl) {
   dieIf.url = apiUrl || dieIf.url;
-  dieIf(err);
+  if (err) return dieIf(err);
```

Comment:
https://github.com/strongloop/strong-mesh-models/blob/cleanup-tests/bin/sl-meshctl.js#L827

This reads funny "if error die if error"... The "if" is what `dieIf()` is already supposed to do. I see your problem, you want to return early. `dieIf()` should renamed `die()` if the `if` always has to be done outside the function.